polaris-react/src/components/Button/Button.scss

```diff
@@ -768,9 +765,7 @@
   }
 
   &.sizeLarge {
-    // stylelint-disable-next-line polaris/conventions/polaris/custom-property-allowed-list -- custom property
-    margin: calc(-1 * var(--pc-button-large-vertical-padding))
-      calc(-1 * var(--p-space-500));
+    margin: calc(-1 * var(var(--p-space-200))) calc(-1 * var(--p-space-500));
```

Comment:
This should also be split out into `margin-block` and `margin-inline` for consistency